### PR TITLE
`Development`: Improve grammar of README.md

### PR DESCRIPTION
**Pull request by bfiics**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [ ] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [ ] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Server
- [ ] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [ ] I added multiple integration tests (Spring) related to the features (with a high test coverage).
- [ ] I added `@PreAuthorize` and checked the course groups for all new REST Calls (security).
- [ ] I implemented the changes with a good performance and prevented too many database calls.
- [ ] I documented the Java code using JavaDoc style.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [ ] Following the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [ ] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/).
- [ ] I added `authorities` to all new routes and checked the course groups for displaying navigation elements (links, buttons).
- [ ] I documented the TypeScript code using JSDoc style.
- [ ] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.
#### Changes affecting Programming Exercises
- [ ] I tested **all** changes and their related features with **all** corresponding user types on Test Server 1 (Atlassian Suite).
- [ ] I tested **all** changes and their related features with **all** corresponding user types on Test Server 2 (Jenkins and Gitlab).

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Log in to Artemis
2. Navigate to Course Administration
3. ...

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Test Coverage

| Class/File | Branch | Line |
|------------|-------:|-----:|

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
    * **Manual reviews** are possible directly in the online code editor or in [Orion](https://github.com/ls1intum/Orion)
-   * **Policies**: Instructors can define submission policies (e.g., penalties after 10 attempts) to prevent that students try out all possibilities without thinking.
+   * **Policies**: Instructors can define submission policies (e.g., penalties after 10 attempts) to prevent students from trying out all possibilities without thinking.
    * **Grading**: Instructors have many options to configure grading, analyze the results based on tests and static code analysis categories and re-evaluate the results
-   * **Secure**: Test cases and student code run in Docker environments on build agents. Test frameworks such as [Ares](https://github.com/ls1intum/Ares) simplify the creation of structural and dynamic tests and prevent that students can cheat. 
+   * **Secure**: Test cases and student code run in Docker environments on build agents. Test frameworks such as [Ares](https://github.com/ls1intum/Ares) simplify the creation of structural and dynamic tests and prevent students from cheating.
```
Comment (Strohgelaender) [inline README.md:20]: For me the old formulation sounds better here (Ares does not prevent cheating attempts, it makes the attempts non sucessfull).

**Review by Strohgelaender — COMMENTED** [review]
Thank you for contributing to Artemis, it's highly apreciated.
The most improvments look good to me, but there us one change where I find the old formulation better:

File: README.md (hunk 1 above)
Comment (joaquinmurcia) [reply README.md:20]: would it be better then "that catch possible student attempts of cheating"?

Comment (Strohgelaender) [reply README.md:20]: No, catch would mean getting notifications or other information about it. Here it is just a security framework that denies attempts.

Comment (manuelmanso) [reply README.md:20]: I disagree, the new formulation sounds better to me.

**Review by manuelmanso — APPROVED** [review]
Improvements looks good to me. Consider updating the checklist in the description, you can simply remove the lines that don't apply, in this case all the lines for "Server" and "Client" can just be removed since you only updated the README

**Review by Mikael31 — APPROVED** [review]
Changes sound good and make the text sound better

**Review by Santia-go — APPROVED** [review]
Improvements look good.

**Review by ge65cer — CHANGES_REQUESTED** [review]
Apart from two tiny changes the PR does not offer any benefits. Also, we should keep the Readme consistent with our About Us page.

**Pull request closed without merging**